src/data/set/pairwise.lean

```diff
@@ -34,7 +34,7 @@ theorem set.pairwise_on.on_injective {s : set α} {r : α → α → Prop} (hs :
   pairwise (r on f) :=
 λ i j hij, hs _ (hfs i) _ (hfs j) (hf.ne hij)
 
-theorem pairwise.mono {p q : α → α → Prop} (h : ∀ ⦃i j⦄, p i j → q i j) (hp : pairwise p) :
+theorem pairwise.mono {p q : α → α → Prop} (hp : pairwise p) (h : ∀ ⦃i j⦄, p i j → q i j) :
```

Comment:
Any reason for swapping these arguments?

Reply:
Dot notation. Dot notation. I only live for dot notation.

Reply:
Dot notation does not care about this

Reply:
It does if you partially apply it, right? I don't get errors here, but I remember trying out something like `lemma convex.is_closed_of_is_compact {A : set α} (is_compact A) (convex A) : is_closed A` and `hA.is_closed_of_is_compact` (where `hA : convex A` didn't work.